Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.Properties.cs

```diff
@@ -139,6 +140,14 @@ public string Text
         }
 
         /// <summary>
+        /// Gets the collection of inline text elements within a Windows.UI.Xaml.Controls.TextBlock.
+        /// </summary>
+        /// <returns>
+        /// A collection that holds all inline text elements from the Windows.UI.Xaml.Controls.TextBlock. The default is an empty collection.</returns>
+        // Hack: I need a InlineCollection from a TextBlock since _textContent is null when XAML accesses the properties
+        public InlineCollection Inlines { get; } = new TextBlock().Inlines;
```

Comment:
Why not use a simple list here, ie. change this line to `public List<Inline> Inlines { get; } = new List<Inline>();`. Then you should be able to avoid the issue you are hitting below.

Reply:
@nmetulev I will try that.

Reply:
@nmetulev Ok this works. Thans for the comment.